public/stylesheets/site/2.0/25-media-midsize.css

```diff
@@ -0,0 +1,47 @@
+/* MEDIA: only screen and (max-width: 64em), handheld ENDMEDIA */
+
+/* 04 region dashboard */
+
+#dashboard {
+  clear: both;
+  float: none;
+  max-width: 100%;
+  margin: 1%;
+  padding: 0;
+  width: auto;
+}
+
+#dashboard .navigation, #dashboard .navigation li, #dashboard a, #dashboard .current {
+  text-align: left;
+  display: inline;
+  border: none;
+}
+
+#dashboard ul {
+  padding: 0;
+}
+
+#dashboard .landmark {
+  clear: none;
+  float: left;
+}
+
+/* 05 region main */
+
+#main, #main.dashboard {
+  float: none;
+  margin: auto;
+  width: auto;
+  padding-left: 0.5em;
+  padding-right: 0.5em;
+}
+
+/* 18 zone searchbrowse */
```

Comment:
Here we increase the width of the filters and decrease the amount of space between the column of blurbs and the filters, mostly so we don't have to have the narrow stylesheet kick in at 48em, but also because it was just kind of weird to have 1% margins on the sides of the page, but a much bigger margin between the blurbs and the filters.